test/integration/main_test.go

```diff
@@ -332,35 +358,35 @@ deletion_timeline:
 	}
 }
 
-// testGetFirstPmapEventWithRetries queries the DB and get the matched PmapEvent.
+// testGetFirstMatchedBQEntryWithRetries queries the DB and get the matched PmapEvent.
```

Comment:
`queries the DB and get the matched PmapEvent`, update it. No more pmapevent here.